### PR TITLE
refactor: rename ovh-angular-responsive-page-switcher

### PR DESCRIPTION
**Pull request by radireddy**
replace module name from ovh-angular-responsive-page-switcher  to ngOvhResponsivePageSwitcher
BREAKING CHANGE: module is now named ngOvhResponsivePageSwitcher

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -1,10 +1,2 @@
...
-.idea
-npm-debug.log
-bower_components
-.work
-.eslintignore
-.editorconfig
-.eslintrc.json
-.*.swp
-example
+githubBanner.png
```
Comment (antleblanc) [inline .npmignore]: We can remove both `githubBanner.png` and `.npmignore` files.

File: README.md [hunk 2]
```diff
@@ -1,78 +1,39 @@
...
-[![Maintenance](https://img.shields.io/maintenance/yes/2018.svg)]() [![Chat on gitter](https://img.shields.io/gitter/room/ovh/ux.svg)](https://gitter.im/ovh/ux) [![Build Status](https://travis-ci.org/ovh-ux/ovh-angular-responsive-page-switcher.svg)](https://travis-ci.org/ovh-ux/ovh-angular-responsive-page-switcher)
-
-[![NPM](https://nodei.co/npm/ovh-angular-responsive-page-switcher.png?downloads=true&downloadRank=true&stars=true)](https://nodei.co/npm/ovh-angular-responsive-page-switcher/)
-
-> `ovh-angular-responsive-page-switcher` module is a set of 2 directives (`responsive-switch` and `responsive-switch-page`) that enable to display content like pages.
-
-It detects available width for displaying page in 2 distinct modes :
+# ng-ovh-responsive-page-switcher
 
+> `ng-ovh-responsive-page-switcher` module is a set of 2 directives (`responsive-switch` and `responsive-switch-page`) that enable to display content like pages. It detects available width for displaying page in 2 distinct modes :
```
Comment (antleblanc) [inline README.md]:
```suggestion
> It detects available width for displaying page in 2 distinct modes
```

File: README.md [hunk 3]
```diff
@@ -1,78 +1,39 @@
...
 * `switch` mode : one page is displayed at a time ;
 * `sidebyside` mode : pages are displayed side by side.
 
-## Table of contents
-* [Dependencies](#dependencies)
-* [Installation](#installation)
-* [Usage and example](#usage-and-example)
-* [Module Components](#module-components)
+[![Downloads](https://badgen.net/npm/dt/@ovh-ux/ng-ovh-responsive-page-switcher)](https://npmjs.com/package/@ovh-ux/ng-ovh-responsive-page-switcher)[![Dependencies](https://badgen.net/david/dep/ovh-ux/ng-ovh-responsive-page-switcher)](https://npmjs.com/package/@ovh-ux/ng-ovh-responsive-page-switcher?activeTab=dependencies)[![Dev Dependencies](https://badgen.net/david/dev/ovh-ux/ng-ovh-responsive-page-switcher)](https://npmjs.com/package/@ovh-ux/ng-ovh-responsive-page-switcher?activeTab=dependencies)[![Gitter](https://badgen.net/badge/gitter/ovh-ux/blue?icon=gitter)](https://gitter.im/ovh/ux)
```
Comment (antleblanc) [inline README.md:5]: Please move badges right after `> It detects...`

File: README.md [hunk 4]
```diff
@@ -1,78 +1,39 @@
...
+
 
 ## Dependencies
```
Comment (antleblanc) [inline README.md]: Could be renamed as `## Related` and moved after `## Test` part.

File: README.md [hunk 5]
```diff
@@ -103,25 +58,12 @@ The example is now running at `http://localhost:7711/example/`.
...
-* Contribute: https://github.com/ovh-ux/ovh-angular-responsive-page-switcher/blob/master/CONTRIBUTING.md
-* Report bugs: https://github.com/ovh-ux/ovh-angular-responsive-page-switcher/issues
-* Get latest version: https://github.com/ovh-ux/ovh-angular-responsive-page-switcher
+Always feel free to help out! Whether it's [filing bugs and feature requests](https://github.com/ovh-ux/ng-ovh-responsive-page-switcher/issues/new) or working on some of the [open issues](https://github.com/ovh-ux/ng-ovh-responsive-page-switcher/issues), our [contributing guide](CONTRIBUTING.md) will help get you started.
 
 ## License
 
-See https://github.com/ovh-ux/ovh-angular-responsive-page-switcher/blob/master/LICENSE
+[BSD-3-Clause](LICENSE) © OVH SAS
+
```
Comment (antleblanc) [inline README.md]: Extra empty line.

File: package.json [hunk 6]
```diff
@@ -1,42 +1,54 @@
 {
-  "name": "ovh-angular-responsive-page-switcher",
+  "name": "@ovh-ux/ng-ovh-responsive-page-switcher",
   "version": "1.1.1",
   "description": "It detects available width for displaying page in 2 distinct modes",
-  "main": "./dist/ovh-angular-responsive-page-switcher.min.js",
-  "repository": "ovh-ux/ovh-angular-responsive-page-switcher",
-  "author": "OVH SAS",
+  "keywords": [
+    "angular",
```
Comment (antleblanc) [inline package.json]: keywords aren't related to this package.

We can use the following list:
- page
- responsive
- switcher
- angularjs
- ovhcloud

File: package.json [hunk 7]
```diff
@@ -1,42 +1,54 @@
...
-    "jshint-stylish": "~1.0.0",
-    "jasmine-core": "^2.6.4",
-    "karma": "~0.13.0",
-    "karma-jasmine": "~0.3.0",
-    "karma-phantomjs-launcher": "~1.0.0",
-    "matchdep": "^0.3.0"
+    "@commitlint/cli": "^8.3.5",
+    "@commitlint/config-angular": "^8.3.4",
+    "@ovh-ux/component-rollup-config": "^7.0.1",
+    "angular-mocks": "~1.7.9",
```
Comment (antleblanc) [inline package.json]: Is `angular-mocks` used?

File: package.json [hunk 8]
```diff
@@ -1,42 +1,54 @@
...
+    "angular-scenario": "~1.7.9",
```
Comment (antleblanc) [inline package.json]: Is `angular-mocks` used?

File: package.json [hunk 9]
```diff
@@ -1,42 +1,54 @@
...
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-formatter-pretty": "^3.0.1",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-markdown": "^1.0.0",
+    "husky": "^4.2.3",
+    "lint-staged": "^10.1.2",
+    "npm-run-all": "^4.1.5",
+    "ovh-angular-responsive-popover": "~5.0.0-alpha.2"
```
Comment (antleblanc) [inline package.json]: Is `ovh-angular-responsive-popover` used?

File: package.json [hunk 10]
```diff
@@ -1,42 +1,54 @@
...
+  },
+  "peerDependencies": {
+    "angular": "^1.7.5",
+    "angular-animate": "^1.5.0",
+    "angular-sanitize": "^1.5.0",
+    "jquery": "~2.1.3",
+    "lodash": "~3.3.0",
```
Comment (antleblanc) [inline package.json]: Lodash must be defined as a `devDependencies`in v4.

File: src/index.js [hunk 11]
```diff
@@ -0,0 +1,17 @@
+import angular from 'angular';
+import 'matchmedia-ng';
+import 'angular-animate';
+
+import responsiveSwitchPage from './ng-ovh-responsive-page-switcher/directive';
+import responsiveSwitch from './ng-ovh-responsive-switch/directive';
```
Comment (antleblanc) [inline src/index.js]:
```suggestion
import directive from './directive';
```

File: src/index.js [hunk 12]
```diff
@@ -0,0 +1,17 @@
...
+
+import './ng-ovh-responsive-page-switcher.less';
+
+const moduleName = 'ngOvhResponsivePageSwitcher';
+
+angular
+  .module(moduleName, ['matchmedia-ng', 'ngAnimate'])
+  .directive('responsiveSwitchPage', responsiveSwitchPage)
+  .directive('responsiveSwitch', responsiveSwitch);
```
Comment (antleblanc) [inline src/index.js]:
```suggestion
  .directive('responsiveSwitch', directive);
```

File: src/index.js [hunk 13]
```diff
@@ -0,0 +1,17 @@
+import angular from 'angular';
+import 'matchmedia-ng';
+import 'angular-animate';
+
+import responsiveSwitchPage from './ng-ovh-responsive-page-switcher/directive';
```
Comment (antleblanc) [inline src/index.js]:
```suggestion
import pageDirective from './page/directive';
```

File: src/index.js [hunk 14]
```diff
@@ -0,0 +1,17 @@
...
+import responsiveSwitch from './ng-ovh-responsive-switch/directive';
+
+import './ng-ovh-responsive-page-switcher.less';
```
Comment (antleblanc) [inline src/index.js]:
```suggestion
import './index.less';
```

File: package.json [hunk 15]
```diff
@@ -1,42 +1,54 @@
...
+    "husky": "^4.2.3",
+    "lint-staged": "^10.1.2",
+    "npm-run-all": "^4.1.5",
+    "ovh-angular-responsive-popover": "~5.0.0-alpha.2"
+  },
+  "peerDependencies": {
+    "angular": "^1.7.5",
+    "angular-animate": "^1.5.0",
+    "angular-sanitize": "^1.5.0",
+    "jquery": "~2.1.3",
```
Comment (antleblanc) [inline package.json]: Is `jquery` used?

**Review by antleblanc — CHANGES_REQUESTED** [review]

File: package.json (hunk 7 above)
Comment (radireddy) [reply package.json]: may be not

File: package.json (hunk 9 above)
Comment (radireddy) [reply package.json]: I did not find any reference for it in the code but it was included previously. Also, below note was there in README.md 
__Note__ : `ng-ovh-responsive-page-switcher` is suitable with [ng-ovh-responsive-popover](https://github.com/ovh-ux/ng-ovh-responsive-popover).

File: package.json (hunk 15 above)
Comment (radireddy) [reply package.json]: it was used, i replaced it with $window

Comment (radireddy) [reply package.json]: maybe it's not required anymore

File: package.json [hunk 16]
```diff
@@ -1,42 +1,53 @@
 {
-  "name": "ovh-angular-responsive-page-switcher",
+  "name": "@ovh-ux/ng-ovh-responsive-page-switcher",
   "version": "1.1.1",
```
Comment (antleblanc) [inline package.json:3]:
```suggestion
  "version": "0.0.0",
```

**Review by antleblanc — APPROVED** [review]

**Pull request merged**